server/app/views/applicant/ApplicantUpsellCreateAccountView.java

```diff
@@ -54,24 +60,35 @@ public Content render(
       Messages messages,
       Optional<ToastMessage> bannerMessage) {
     boolean shouldUpsell = shouldUpsell(account);
-
+    String redirectUrl = routes.UpsellController.download(applicationId, applicantId).url();
     Modal loginPromptModal =
         createLoginPromptModal(
                 messages,
                 redirectTo,
                 /* description= */ messages.at(MessageKey.GENERAL_LOGIN_MODAL_PROMPT.getKeyName()),
                 /* bypassMessage= */ MessageKey.BUTTON_CONTINUE_WITHOUT_AN_ACCOUNT)
             .build();
-
+    ATag downloadButton = new ATag();
+    if (settingsManifest.getApplicationExportable(request)) {
+      downloadButton =
+          new ATag()
+              .withHref(redirectUrl)
+              .with(
+                  button("Download PDF")
```

Comment:
**suggestion (blocking):** this hard-coded UI string (and any others added in this PR) need to be added to the i18n system